packages/driver/cypress/e2e/commands/aliasing.cy.js

```diff
@@ -29,6 +29,11 @@ describe('src/cy/commands/aliasing', () => {
       })
     })
 
+    it('stores the lookup as an alias when .then() is an intermediate', () => {
```

Comment:
This was the minimal repro of the issue causing the todomvc failure, included in the monorepo to prevent any future regression. Explanation about what was going wrong and how it was fixed in another comment.